fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/event/AbstractEventCreationFragment.java

```diff
@@ -36,6 +36,7 @@
   public static final String NO_LOCATION = "";
   public final DateFormat DATE_FORMAT = new SimpleDateFormat("dd/MM/yyyy", Locale.FRENCH);
   public final DateFormat TIME_FORMAT = new SimpleDateFormat("HH:mm", Locale.FRENCH);
+  public static final long creationTimeInSeconds = Instant.now().getEpochSecond();
```

Comment:
As sonar suggests: isn't it more appropriate to use an UPPER_CASE variable here ?